dashboard/test/controllers/user_levels_controller_test.rb

```diff
@@ -92,4 +92,71 @@ class UserLevelsControllerTest < ActionController::TestCase
 
     refute_nil UserLevel.find_by(id: user_level.id)
   end
+
+  [Multi, FreeResponse].each do |level_type|
```

Comment:
These tests are 🔥 for documentation.

Just to triple check: A teacher can delete/reset their own progress only –– students can't do this, teachers can't delete other teachers' progress, and teachers can't delete their own students' progress.

Reply:
That's correct, at least for now. There's definitely a desire for students to delete their own progress in certain cases or for teachers to delete their students' progress, but we're not there yet